src/pages/guidelines/2x-grid/code.mdx

```diff
@@ -16,9 +22,32 @@ tabs: ["Basics", "Implementation"]
 
 </AnchorLinks>
 
-## Under the hood
+## Resources
```

Comment:
Would we also want to link to the React storybook here? https://react.carbondesignsystem.com/?path=/story/grid--auto-columns

Reply:
👍